.golangci.yml

```diff
@@ -132,6 +131,9 @@ issues:
   - linters:
     - gosec
     text: "G304: Potential file inclusion via variable"
+  - linters:
+    - revive
+    text: "package-comments: should have a package comment"
```

Comment:
If we want to add package comments everywhere I would suggest to do that in follow-up PRs (I can open an issue for it).

There are a lot of packages with missing package comments